backend/src/openarchiefbeheer/destruction/tests/endpoints/test_destruction_list_items.py

```diff
@@ -30,7 +33,16 @@ def test_not_authenticated(self):
 
         self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
 
-    def test_retrieve_destruction_list_items(self):
+    @requests_mock.Mocker()
```

Comment:
Maybe instead of the mock, you could use the feature flag to disable the count?

Reply:
I agree that we need a test for the feature flag but maybe not use the feature flag to hide the behavior for convenience, using a mock we at least test some of the machinery?